docs/hazmat/primitives/asymmetric/ec.rst

```diff
@@ -20,6 +20,22 @@ Elliptic curve cryptography
     :returns: A new instance of :class:`EllipticCurvePrivateKey`.
 
 
+.. function:: derive_private_key(secret, curve, backend)
+
+    .. versionadded:: 1.5.4
+
+    Derive a private key from ``secret`` on ``curve`` for use with ``backend``.
+
+    :param secret: An instance of :class:`int`.
```

Comment:
This can be `:param int secret: The secret scalar value` (or some better description of what secret really is)